src/node_url.cc

```diff
@@ -131,11 +131,13 @@ namespace url {
   static int ToUnicode(std::string* input, std::string* output) {
     output->reserve(input->length());
     *output = input->c_str();
```

Comment:
Good point – if I’m reading the code correctly, this *should* be `*output = *input`? At least for `ToASCII`, there’s an explicit check for null bytes just below the call to it.